omero/sysadmins/cli/data-ownership.txt

```diff
@@ -0,0 +1,53 @@
+Changing the ownership of data
```

Comment:
This also talks about moving data between groups which isn't obviously about ownership - maybe it would be clearer to call this 'Data management' to match the 'User/group management' title also used in the admin section.

That also means you could instead use this title or one more like it for the user page